uid2_client/input_util.py

```diff
@@ -1,6 +1,8 @@
 import hashlib
 import base64
 
+import pytz
```

Comment:
Nope, we need to remove the timezone, or it will throw an exception HTTP Error 400: Bad Request.

Reply:
Right, but I'm talking about the line that converts the timestamp to UTC, not the line that removes the timezone (`dt_utc_without_tz = dt_utc.replace(tzinfo=None)`).

I'm talking about replacing

```
dt_utc = timestamp.astimezone(pytz.utc)
```

with

```
dt_utc = timestamp.astimezone(timezone.utc)
```

so that we don't need to add a new dependency.